internal/resources/resource.go

```diff
@@ -100,3 +100,11 @@ func CheckObjectUpdatedIgnoreStatus(current, updated runtime.Object) (bool, erro
 	}
 	return !patchResult.IsEmpty(), nil
 }
+
+func CopyDict(src map[string]string) map[string]string {
+	dst := make(map[string]string)
```

Comment:
you can add `len(src)` for map creation, something like

```
dst := make(map[string]string, len(src))
```
https://go.dev/play/p/sOLFbl_EHuj

Reply:
Ok, that's better